python/tests/table/test_partitioning.py

```diff
@@ -29,8 +30,37 @@ def test_partition_field_init():
     assert partition_field.transform == bucket_transform
     assert partition_field.name == "id"
     assert partition_field == partition_field
+    print(str(partition_field))
+    print("repr")
+    print(repr(partition_field))
```

Comment:
Can you remove all print statements from the tests?

Reply:
Thank you, I was testing and forgot about this, removed